.circleci/config.yml

```diff
@@ -62,6 +62,7 @@ workflows:
             tags:
               only: /^v.*$/
       - docker_push_tag:
+          context: ae_dockerhub
```

Comment:
`_` or `-`?

Reply:
From https://circleci.com/gh/organizations/aeternity/settings#contexts :
```suggestion
          context: ae-dockerhub
```

Reply:
Ah, this is what happens in a rush. Thanks for noticing it @lucafavatella